onnx/optimizer/passes/fuse_add_bias_into_conv.h

```diff
@@ -0,0 +1,128 @@
+// ATTENTION: The code in this file is highly EXPERIMENTAL.
+// Adventurous users should note that the APIs will probably change.
+
+#pragma once
+
+// Z = Conv(X, Y)
+// B = Z + A
```

Comment:
can you add a line that describes what this is transformed into? I guess it would be `B = Conv(X, Y, A)` or something

Reply:
Good point :-)